plugins/toolkit/jetbrains-core/tst/software/aws/toolkits/jetbrains/services/codewhisperer/codemodernizer/CodeWhispererCodeModernizerTest.kt

```diff
@@ -47,44 +37,6 @@ class CodeWhispererCodeModernizerTest : CodeWhispererCodeModernizerTestBase() {
         super.setup()
     }
 
-    @Test
-    fun `test runModernize handles user cancelled dialog`() {
-        doReturn(null).whenever(codeModernizerManagerSpy).getCustomerSelection(any())
-        runInEdtAndWait {
-            val buildFiles = listOf(emptyPomFile)
-            val job = codeModernizerManagerSpy.runModernize(buildFiles)
-            assertNull(job)
-            verify(codeModernizerManagerSpy, times(1)).runModernize(buildFiles)
-            verify(codeModernizerManagerSpy, times(1)).getCustomerSelection(buildFiles)
-            // verify execution stopped after user cancelled
-            verify(codeModernizerManagerSpy, times(0)).createCodeModernizerSession(any(), any())
-            verify(codeModernizerManagerSpy, times(0)).launchModernizationJob(any())
-            verify(codeModernizerManagerSpy, times(0)).postModernizationJob(any())
-            verifyNoMoreInteractions(codeModernizerManagerSpy)
-        }
-    }
```

Comment:
Blocker, Question: 
I am seeing a lot of changed / deleted tests but not any added tests. Is there any testing we can do to ensure that the main flow works as expected? No need for the UI to be tested in this pr as its complicated to my understanding but probably worthwhile to add tests for the logical flow?

Reply:
The removed tests were mainly for the removed button actions.

I agree that we should add tests for logical flows. I can take it as a follow up.